veba-bom.json

```diff
@@ -17,136 +17,138 @@
         }]
     },
     "antrea": {
-        "gitRepoTag": "v1.10.0",
+        "gitRepoTag": "v1.15.0",
         "containers": [{
-            "name": "projects.registry.vmware.com/antrea/antrea-ubuntu",
-            "version": "v1.10.0"
-        }]
+                "name": "projects.registry.vmware.com/antrea/antrea-agent-ubuntu",
+                "version": "v1.15.0"
+            },
+            {
+                "name": "projects.registry.vmware.com/antrea/antrea-controller-ubuntu",
+                "version": "v1.15.0"
+            }
+        ]
     },
     "fluentbit": {
-        "gitRepoTag": "v2.0.8",
+        "gitRepoTag": "v2.2.2",
         "containers": [{
                 "name": "fluent/fluent-bit",
-                "version": "2.0.8"
+                "version": "2.2.2"
             }]
     },
     "cadvisor": {
-        "gitRepoTag": "v0.47.0",
+        "gitRepoTag": "v0.47.2",
         "containers": [{
                 "name": "gcr.io/cadvisor/cadvisor",
-                "version": "v0.47.0"
+                "version": "v0.47.2"
             }]
     },
     "kind": {
         "gitRepoTag": "main",
         "containers": [{
             "name": "kindest/node",
-            "version": "v1.25.3"
+            "version": "v1.29.1"
         }]
     },
     "kubernetes": {
-        "gitRepoTag": "v1.25.5",
+        "packageRepoTag": "v1.29",
```

Comment:
Let's not introduce a new key here, given the difference between `packageRepoTag` and `gitRepoTag` is just removing the last octet, lets do that in shell script itself

Reply:
Done